packages/cli/src/index.ts

```diff
@@ -209,6 +209,11 @@ yargs
           array: true,
           type: 'string',
         },
+        'clean-components': {
```

Comment:
I would recommend this change to `remove-unused-components`. It used in a few different ways in this PR "clear components" and "clean components" and "clean unused components" and I think it should be standardized on "remove-unused-components".

Reply:
@adamaltman done